README.md

```diff
@@ -22,7 +22,7 @@ as described in the [documentation](https://docs.opensafely.org/getting-started/
 If running it directly, it should be run from within the research repository.
 To run the latest version via Docker and access its full help:
 
-    docker run --rm ghcr.io/opensafely-core/cohortextractor-v2 --help
+    docker run --rm ghcr.io/opensafely-core/databuilder --help
```

Comment:
thought: Maybe we shouldn't advocate the latest version and pin to a specific version here, e.g. v0 for now?

Reply:
Actually, going to leave this as is for now, because the new image name won't work yet.